README.md

```diff
@@ -392,12 +392,23 @@ Third argument (optional) is entire payload from Google API
 ```js
 import { geocodeByAddress } from 'react-places-autocomplete'
 
+// Callback use
 geocodeByAddress('Los Angeles, CA', (error, { lat, lng }, results) => {
   if (error) { return }
 
   console.log('Geocoding success!', { lat, lng })
   console.log('Entire payload from Google API', results)
 })
+
+// Promise use
+geocodeByAddress('Los Angeles, CA')
+  .then(({ { lat, lng }, results }) => {
+    console.log('Geocoding success!', { lat, lng })
+    console.log('Entire payload from Google API', results)
+  })
+  .catch(() => {
+    return

```

Comment:
@jxom Let's provide a simple example to demonstrate how to use the error object passed from the Promise `reject` in the README 👍